setup.py

```diff
@@ -113,8 +114,8 @@ def find_version(*file_paths):
     "tests": ["pytest"],
     "docs": [],
     "tf": [
-        "tensorflow~=2.5.0",
-        "numpy~=1.19.2",
+        "tensorflow~=2.8.0",
+        "numpy~=1.20, <1.23.0",
```

Comment:
```suggestion
        "numpy>=1.19.2",
```
to support the previous Tensorflow version

Reply:
Actually there's version mismatch(tf 2.8 requires numpy >= 1.20), and <1.23.0 is for scipy. Maybe the numpy version should be divided by 1.20 depending on the tf version.

Reply:
Thanks for the reply! What version does scipy require numpy<1.23.0?

> Maybe the numpy version should be divided by 1.20 depending on the tf version.
Could you give more context on what you mean?

Reply:
> Thanks for the reply! What version does scipy require numpy<1.23.0?

When install numpy >= 1.23.0, below error occurred.

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
nncf 2.3.0.dev0+editable requires numpy<1.23.0,~=1.20, but you have numpy 1.23.0 which is incompatible.
scipy 1.7.3 requires numpy<1.23.0,>=1.16.5, but you have numpy 1.23.0 which is incompatible.
```

Our scipy version in current `setup.py` requires to install <1.8 in `setup.py`, so I added numpy < 1.23.0.

> Could you give more context on what you mean?

Sure! What I mean was depending on the TF version, the numpy version should be divided more than 1.20 or lower than 1.20.
if TF >= 2.8.0 then Numpy >= 1.20
if TF <= 2.6.* then Numpy < 1.20
in TF 2.7.*, it seems like support broader range of numpy.

Reply:
The scipy version was restricted from the top to avoid installation of the release candidate instead of the stable version. It looks like we can bump up the scipy version to the latest stable version 1.8.1.
```
"scipy>=1.3.2, <1.9; python_version>='3.7'",
```
I hope It can solve the issue with numpy.
 
> Sure! What I mean was depending on the TF version, the numpy version should be divided more than 1.20 or lower than 1.20.
> if TF >= 2.8.0 then Numpy >= 1.20
> if TF <= 2.6.* then Numpy < 1.20
> in TF 2.7.*, it seems like support broader range of numpy.

Thanks, the idea is clear! I would like to look at the implementation.

Reply:
Scipy and numpy ver relaxation would be good. I'll test and see if it is available.
I'll see also the numpy version relaxation for TF.